tests/e2e-kubernetes/testsuites/credentials.go

```diff
@@ -1041,6 +1046,28 @@ func createRole(ctx context.Context, f *framework.Framework, assumeRolePolicyDoc
 		framework.ExpectNoError(err)
 	}
 
+	framework.Gomega().Eventually(ctx, framework.HandleRetry(func(ctx context.Context) (bool, error) {
```

Comment:
Should we also add a log line before this line to say "Waiting until all policies are attached"?